### PR TITLE
CNV-8424: Add Drawer with interface basic info

### PR DESCRIPTION
**Pull request by upalatucci**
Drawer on interface click

Drawer has 'Details' and 'YAML' menu

I was not able to use the HorizontalNav as it requires completely changing the URL on click (ex. from `/LISTURL/yaml` to `/LIST_URL/details`) and reloading the page. 

But the user does not change the page. So I copied the classes to have the same design. 
An alternative would be to use the Patternfly Tabs that do not have the same UI. 

Screenshot from 2023-06-14 10-31-44
Screenshot from 2023-06-14 10-31-38
Screenshot from 2023-06-14 10-31-32

### CODE REVIEW

**Comment by pcbailey** [comment]
You have some lint warnings that you need to take care of. Check the commit screen.

**Comment by upalatucci** [comment]
@pcbailey new cypress github action version fails ....

**Comment by upalatucci** [comment]
fix is here https://github.com/nmstate/nmstate-console-plugin/pull/40

**Review by hstastna — COMMENTED** [review]
There's bunch of issues in `InterfaceDrawer.tsx` file. PTAL ;) 
ptal

**Comment by upalatucci** [comment]
@hstastna i've finally managed to configure eslint with the `no-unused-vars` rule.

File: src/views/states/list/components/InterfaceDrawer/InterfaceDrawerDetailsTab.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+
+import { Checkbox, Flex, FlexItem, List, ListItem, Title } from '@patternfly/react-core';
+import { NodeNetworkConfigurationInterface } from '@types';
+
+type InterfaceDrawerDetailsTabProps = {
+  selectedInterface: NodeNetworkConfigurationInterface;
+};
+
+const InterfaceDrawerDetailsTab: React.FC<InterfaceDrawerDetailsTabProps> = ({
```
Comment (pcbailey) [inline src/views/states/list/components/InterfaceDrawer/InterfaceDrawerDetailsTab.tsx]: Please import `FC`. =)

File: src/views/states/list/components/InterfaceDrawer/InterfaceDrawerYAMLTab.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { dump } from 'js-yaml';
+
+import { NodeNetworkConfigurationInterface } from '@types';
+
+type InterfaceDrawerYAMLTabProps = {
+  selectedInterface: NodeNetworkConfigurationInterface;
+};
+
+const InterfaceDrawerYAMLTab: React.FC<InterfaceDrawerYAMLTabProps> = ({ selectedInterface }) => (
```
Comment (pcbailey) [inline src/views/states/list/components/InterfaceDrawer/InterfaceDrawerYAMLTab.tsx]: Please import `FC`. =)

**Pull request merged**